src/lib/pickles/wrap_verifier.ml

```diff
@@ -1,67 +1,60 @@
-module type Inputs = Intf.Wrap_main_inputs.S
-
 module S = Sponge
-open Backend
-open Core_kernel
```

Comment:
Why remove this open? I think we want to default to opening it, so that we don't accidentally use the non-tail-recursive `List.map` and friends.

Reply:
This one is indeed debatable. 

 If you *always* want to have `Core_kernel` open, the best way to do it is to have a flag in `dune` that state `-open Core_kernel`. Should I just add a commit on top of this series that does that?

Nit: `List.map` is tail-rec these days in the standard library and probably the friends you're alluding to ;-)

Reply:
Also: shouldn't we be using `Core`? (see https://github.com/janestreet/core_kernel/blob/master/core_kernel/src/core_kernel.ml)

Reply:
The last commit pushed 78a4bc045e opens `Core_kernel` by default.

Reply:
It would be better not to make unrelated changes as part of this (already large) PR. Ideally this would just add the interfaces